checklist/README.md

```diff
@@ -0,0 +1,71 @@
+# AMP Contribution Checklist Bot
+
+A GitHub app which posts checklists on pull requests.
+
+This app runs as a Google Cloud Functions deployment. It can be configured for deployment to any organization.
+
+## Interface
+
+The AMP Contribution Checklist Bot watches for all newly opened pull requests.
+
+If the pull request includes a new directory in `extensions/`, the bot will:
+
+1. edit the pull request's description to insert a checklist
+2. post a comment on the pull request metioning the added checklist
+
+## Webhooks
+
+The app subscribes to the following GitHub Webhooks:
+
+- [`PullRequestEvent`](https://developer.github.com/v3/activity/events/types/#pullrequestevent)
+  - `opened`: check the PR content and its comments to post checklist
+
+## Setup
+
+1. Clone this repository and `cd` into the `checklist` directory.
+2. `npm install`
+3. Start a new [Smee channel](https://smee.io/). This can be used to proxy
+   GitHub webhooks to your local machine.
+4. Create a new [GitHub App](https://github.com/settings/apps/new) with the following settings:
+   - General
+     - Set _Homepage URL_ to the repository URL
+     - Set _Webhook URL_ to the Smee channel (development) or the Google Cloud Function URL `https://[REGION]-[PROJECT_NAME].cloudfunctions.net/[FUNCTION_NAME]` (production)
+     - Set _Webhook Secret_ to any high-entropy string of your choice
+   - Permissions and Events
+     - Repository permissions
+       - Set _Contents_ to Read
+       - Set _Pull requests_ to Read & write
+     - Subscribe to events: _Pull request_
+5. After creating the application, generate and download a private key. Also
+   take note of the App ID and app Client Secret.
+6. Install the application on a GitHub repository that you want to use for
+   testing. You might want to fork the [ampproject/amphtml](https://github.com/ampproject/amphtml) repository or create a new repository for this purpose.
+7. Copy the `redacted.env` file to `.env` and modify the fields based the values from the GitHub App page:
+   - The value for the `APP_ID` field is the App ID from Step 5.
+   - The value for `WEBHOOK_SECRET` is the secret you set when creating the GitHub app.
+   - The value for the `PRIVATE_KEY` field is a base64 representation of the
+     `.pem` file you downloaded from the GitHub App page in Step 5. On Linux/Mac you can convert that file by running `cat private-key-file.pem | base64` in a command line.
+   - The value for `GITHUB_ORG` to the name of the GitHub organization the bot will be inviting users to.
```

Comment:
This line is from invite-bot, and isn't relevant here. You'll probably want `GITHUB_ORG` and `GITHUB_REPO`

Reply:
Oh, yeah the description is wrong.

How about we remove both `GITHUB_` environment variables altogether? The webhook context has both the `owner` and the `repo` name, we can just use those rather than setting them statically.